onnxruntime/python/tools/quantization/registry.py

```diff
@@ -80,7 +80,7 @@ def CreateOpQuantizer(onnx_quantizer, node):
     return QuantOperatorBase(onnx_quantizer, node)
 
 
-def CreateQDQQuantizer(onnx_quantizer, node):
+def CreateQDQQuantizer(onnx_quantizer, node, disable_qdq_for_node_output):
```

Comment:
In this PR, we can disable or enable output quantization for all nodes. But in the future we may need to do it on a node to node basis. for example, don't quantize output for weighted ops such as conv, gemm and matmul. But for other node we may or may not quantize the outputs, depending on their successor nodes and different scenario.

Reply:
Yes, good point, I think we can add a member "nodes_to_exclude_output_quantization" to ONNXQuantizer for us to specify output quantization on node basis.

Reply:
if output is not quantized, then the node/op can not be fused into quantized operator. If so, why don't we just exclude the node from quantization?

Reply:
You don't need to touch each operators. It is easy to find the set of nodes, which produce graph model output. Then you can exclude them from quantization.

Reply:
> if output is not quantized, then the node/op can not be fused into quantized operator. If so, why don't we just exclude the node from quantization?
Here we are only handling inserting QDQ to node's inputs/outputs and not quantizing the node itself.
For a node, we can only adding QDQ for its inputs but not adding QDQ for its outputs. So we can't exclude the node from quantization.

Reply:
> if output is not quantized, then the node/op can not be fused into quantized operator. If so, why don't we just exclude the node from quantization?

In TRT, it's not recommended to quantize outputs for weighted ops such as Conv, Matmul, Gemm because those ops may be followed by nodes that requires high resolution inputs. TRT can still fuse DQ+the op, but keep the output as floating point type.

Reply:
You don't need to pass the option in the constructor of operator. Each operator can read the info from self.quantizer